# A Gaussian mixture model-based variational graph autoencoder (GMM-VGAE) algorithm for clustering single-cell RNA-seq data
## Abstract
Cell type identification from single-cell RNA sequencing (scRNA-seq) is crucial to understand disease mechanisms for disease diagnosis and drug discovery, which involves classifying the data into clusters of single cells. scRNA-seq data is high dimensional with many analysis challenges. In this study, we propose to integrate advanced statistical modelling and deep learning and develop a Gaussian mixture model-based variational graph autoencoder (GMM-VGAE) to address this important issue, which is an unsupervised clustering algorithm that takes inputs of cell graph adjacency and gene feature matrix to feed through the graph encoder and decoder to reconstruct the adjacency matrix. We applied the method and other three baseline models to cluster the single cells from three publicly available data sets. Using adjusted rand index as a model performance measure, we showed the GMM-VGAE algorithm has better performance than other baseline models in all three data sets. By incorporating the GMM-VGAE algorithm in scRNA sequencing, the performance of future analysis and application can be enhanced.